operator/docker/files/plot-results.py

```diff
@@ -0,0 +1,300 @@
+#!/usr/bin/env python3
+
+import numpy as np
+import pandas as pd
+import sys
+import os
+import matplotlib.pyplot as plt
+
+# Dictionary with nodes' names and corresponding IP addresses
+nodes_dict = {
+    'Node light': '172.16.238.10',
+    'Node mid': '172.16.238.11',
+    'Node heavy': '172.16.238.12',
+}
```

Comment:
Not sure whether these IPs should be hardcoded

Reply:
The node IPs are also hardcoded in [`dfaas/operator/dfaasoperator.env`](https://github.com/unimib-datAI/dfaas/blob/e4a3f73506f36598cd0b9e7553088da016c26f75/operator/dfaasoperator.env), which is the configuration file of the operator. I modified the Python file to read the IPs from the configuration file (actually the env variable, because the operator is run as a container and the conf file is passed as env variables).